pom.xml

```diff
@@ -132,6 +128,13 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>com.google.cloud</groupId>
+                <artifactId>libraries-bom</artifactId>
+                <version>26.11.0</version>
```

Comment:
Instead of adding version explicitly here - lets create a variable in properties section and use that here.